homeassistant/util/dt.py

```diff
@@ -229,6 +229,8 @@ def parse_time_expression(parameter: Any, min_value: int, max_value: int) -> Lis
     elif isinstance(parameter, str) and parameter.startswith("/"):
         parameter = int(parameter[1:])
         res = [x for x in range(min_value, max_value + 1) if x % parameter == 0]
+    elif isinstance(parameter, str) and parameter.find(",") >= 0:
+        res = [int(x) for x in parameter.split(",") if x]
     elif not hasattr(parameter, "__iter__"):
         res = [int(parameter)]
     else:
```

Comment:
It looks like `parse_time_expression` can handle lists. So we could avoid adding this new logic and instead update the validation to allow a YAML list to achieve the same result:

```
automation:
  trigger:
    platform: time_pattern
    hours: 8
    minutes:
      - 10
      - 15
    seconds: 0

```

Reply:
Indeed, it looks so. If frontend allows to define list (ie. like `[20,21,22]`) then we're home ;)
So should I try to do this?